### PR TITLE
Add the ability to manage dependent packages else where.

### PR DESCRIPTION
**Pull request by kwolf**
We already manage one of the packages (`ca-certificates`) elsewhere.  This adds the ability to manage dependent packages outside of the nodejs module to avoid duplicate package definitions.

### CODE REVIEW

**Comment by juniorsysadmin** [comment]
@kwolf Would prefer that this uses `manage_dependencies` as the parameter for semi-consistent variable naming reasons

**Comment by kwolf** [comment]
I thought about that but I wanted it to be more clear that it was package dependencies.  I am still happy to change it if you want me too.

**Comment by juniorsysadmin** [comment]
Puppet will pull in package dependencies via yum, apt etc so leaving package in the name might lead to confusion. ```manage_repo_dependencies``` is even more explicit, if you were after more naming suggestions...

File: README.md [hunk 1]
```diff
@@ -469,6 +469,12 @@ that were available on 2017-01-08:
 The USE flags to use for the Node.js package on Gentoo systems. Defaults to
 ['npm', 'snapshot'].
 
+#### `manage_dep_packages`
```
Comment (juniorsysadmin) [inline README.md]: Make it clear in the README that this refers to requirements for an apt HTTPS repository, not nodejs itself.

**Comment by juniorsysadmin** [comment]
@kwolf Status of this PR?

**Comment by juniorsysadmin** [comment]
Kinda superseded by #348 . Which might make it more difficult to solve your problem. Or not.

**Pull request closed without merging**